### PR TITLE
fix(createIcon): Spread className to svg to fix broken images

### LINKED ISSUE #8936: Bug - Card demo -
**Describe the problem**
A clear and concise description of the problem. Which components are affected?
The icon is not displaying correctly in the "Utilization Card 3 demo"  https://patternfly-react-v5.surge.sh/components/card/react-demos#

image

**How do you reproduce the problem?**
Navigate to the demo https://patternfly-react-v5.surge.sh/components/card/react-demos#
the icon appears larger that it is supposed to.  See the c[ore demo](https://pf5.patternfly.org/components/card/html-demos#utilization-card-3) or how it should appear.

### LINKED ISSUE #8947: Bug - [EmptyState] - [EmptyStateIcon is enormous no matter the EmptyState components variant]
**Describe the problem**
Follows a screenshot of the PF5 docs I build locally.

**Expected behavior**
EmptyStateIcon should follow the Icon sizing and derive from the EmptyState component variant it's sizing.

**Is this issue blocking you?**
Custom CSS can be applied I guess as an override.

**Screenshots**
Screenshot from 2023-04-13 12-23-45

**What is your product and what release date are you targeting?**
Cockpit

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #8936

**Additional issues**: Closes #8947  

I updated the jest version.  There was an issue with snaphots adding undefined attributes that was failing some tests.

### CODE REVIEW

File: packages/react-core/src/components/HelperText/__tests__/__snapshots__/HelperText.test.tsx.snap [hunk 1]
```diff
@@ -103,7 +103,7 @@ exports[`HelperText icon helper text renders properly 1`] = `
       >
         <svg
           aria-hidden="true"
-          class="pf-svg"
+          class="pf-svg undefined"
```
Comment (srambach) [inline packages/react-core/src/components/HelperText/__tests__/__snapshots__/HelperText.test.tsx.snap:106]: Is the `undefined` class supposed to be there?

Comment (tlabaj) [reply packages/react-core/src/components/HelperText/__tests__/__snapshots__/HelperText.test.tsx.snap:106]: I don't t see it on the actual component in docs.  I added a check to only spread the className when not `undefined`  I am not sure why it is happening with snapshots.

Comment (tlabaj) [reply packages/react-core/src/components/HelperText/__tests__/__snapshots__/HelperText.test.tsx.snap]: Ok, I updated our version of jest.  I was seeing different results in CI vs in our workspace.  Updating jest fixed it.

**Review by gitdallas — APPROVED** [review]

File: packages/react-icons/src/createIcon.tsx [hunk 2]
```diff
@@ -32,14 +33,15 @@ export function createIcon({
     id = `icon-title-${currentId++}`;
 
     render() {
-      const { title, ...props } = this.props;
+      const { title, className, ...props } = this.props;
+      const classes = className ? `pf-svg ${className}` : "pf-svg";
```
Comment (jenny-s51) [inline packages/react-icons/src/createIcon.tsx:37]: 👌

**Review by jenny-s51 — APPROVED** [review]
LGTM @tlabaj - empty states are looking good, svg sizes as expected

**Review by srambach — APPROVED** [review]

**Comment by tlabaj** [comment]
> @tlabaj Broken images seem to be fixed, but there is still an issue with Card View not displaying a full grid of cards.
> 
> Screenshot 2023-04-25 at 5 00 02 PM

I just took a look at this and it is not related to the `createIcon`.  For some reason the fetch of the mock data is not working.
I opened issue https://github.com/patternfly/patternfly-react/issues/9003 to address this

**Review by mcarrano — APPROVED** [review]
> I opened issue https://github.com/patternfly/patternfly-react/issues/9003 to address this

Thanks @tlabaj . In that case, I approve.

**Pull request merged**